components/schema-migrator/run.sh

```diff
@@ -75,3 +77,5 @@ if [[ "${NON_INTERACTIVE}" == "true" ]]; then
 else
     $CMD
 fi
+
+set +e
```

Comment:
usually this is put on the top of the script. Or is there any particular reason  to put it in the bottom ?

Reply:
I wasn't sure if once setting `set -e` the effect would stay for parent scripts calling this one and so I added this just in case. If you think it's unnecessary, I can drop it or change it somehow.